packages/core/src/web/bridges/hooks/use-bridge.ts

```diff
@@ -56,6 +65,40 @@ const getExternalStore = <K extends keyof BridgeInterface>(
       },
     };
   }
+  if (key === "userAgent") {
+    return {
+      subscribe: (onChange) => {
+        const deviceCapabilitiesUnsubscribe =
+          bridge.subscribe("deviceCapabilities")(onChange);
+        const platformUnsubscribe = bridge.subscribe("platform")(onChange);
+
+        return () => {
+          deviceCapabilitiesUnsubscribe();
+          platformUnsubscribe();
+        };
+      },
```

Comment:
I'd instead update subscribe interface to accept an array of values. Please use an overloading to avoid having to change syntax on all subscribe calls that currently uses only a single string argument :)

```suggestion
      subscribe: bridge.subscribe(["deviceCapabilities", "platform"])
```